components/bench/ycsb.go

```diff
@@ -139,11 +139,11 @@ func registerYcsb(root *cobra.Command) {
 	cmd.PersistentFlags().Uint32Var(&config.ConnCount, "conncount", 128, "Connection Count")
 	cmd.PersistentFlags().BoolVar(&config.ReadAllFields, "readallfields", true, "Whether Read All Fields")
 
-	cmd.PersistentFlags().Float32Var(&config.ReadProportion, "readproportion", 0, "What proportion of operations are reads")
-	cmd.PersistentFlags().Float32Var(&config.UpdateProportion, "updateproportion", 0, "What proportion of operations are updates")
-	cmd.PersistentFlags().Float32Var(&config.ScanProportion, "scanproportion", 0, "What proportion of operations are scans")
-	cmd.PersistentFlags().Float32Var(&config.InsertProportion, "insertproportion", 0, "What proportion of operations are inserts")
-	cmd.PersistentFlags().Float32Var(&config.ReadModifyWriteProportion, "readmodifywriteproportion", 0, "What proportion of operations are read-modify-write")
+	cmd.PersistentFlags().Float32Var(&config.ReadProportion, "readproportion", 0, "What proportion of operations are reads, default 0.95")
```

Comment:
```suggestion
	cmd.PersistentFlags().Float32Var(&config.ReadProportion, "readproportion", 0.95, "What proportion of operations are reads, default 0.95")
```

Reply:
Actually, I want to remind users of the default value of workload in `help`.
If I change it like this, users won't know it in the `help` like before.
BTW, we needn't change it explicitly, the default workload in YCSB is 0.95/0.05

Reply:
> 

ok, maybe you also need to modify the defaults that actually run ?

Reply:
done